netci.groovy

```diff
@@ -20,7 +20,7 @@ def addBuildStepsAndSetMachineAffinity(def job, String os, String configuration)
     steps {
       if (os == "Windows_NT") {
         batchFile("git submodule update --init --recursive");
-        batchFile(".\\build.cmd /p:Configuration=${configuration} ${loggingOptions}")
+        batchFile(".\\build.cmd /p:Configuration=${configuration} ${loggingOptions} /p:RootRepo=core-setup")
```

Comment:
We customize root repo for ARM in dir.props, could you do that for this case too to avoid the CI modfication? (This also lets us avoid having to change it back later, and makes CI results clearer during the change.)

https://github.com/dotnet/source-build/blob/ddbaffcc8e02cc55c643513bf6c251b06d2b30cf/dir.props#L99-L100